docker-compose.yml

```diff
@@ -69,5 +69,13 @@ services:
     volumes:
       - "postgres_data:/var/lib/postgresql/data"
     restart: "unless-stopped"
+  #mysql:
```

Comment:
how the other containers will identify this service? maybe we need to use a generic `db` service?

Reply:
I was thinking of changing the container name to db, but wasn't sure if it would break anything for existing users.